### DISCUSSION TITLE
⏳ 🔗 Allow to select native Post Custom Fields option in Dynamic Tags

### DISCUSSION THREAD

**Discussion by roboes** [elementor/elementor#27471] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

WordPress 6.5 introduced the [Custom Fields API](https://make.wordpress.org/core/2023/12/07/roadmap-to-6-5/) and WooCommerce 8.9 introduced the [Custom Fields in the New Product Editor](https://developer.woocommerce.com/2024/05/14/woocommerce-8-9-new-product-editor-more-customizable-than-ever/).

Elementor's "Display Conditions" already displays Custom Fields without requiring Advanced Custom Fields (ACF) or similar:

![1](https://github.com/elementor/elementor/assets/44812635/44b0a491-893a-4594-9ae4-af8cc159ce63)

It would be great to see in the "Dynamic Tags" the option to select Custom Fields without requiring any plugin too:

![2](https://github.com/elementor/elementor/assets/44812635/f4f20905-fa3c-4960-8306-734fe53d376a)

### Describe the solution you'd like.

I would like to have the option to show Custom Fields without a plugin (e.g. Advanced Custom Fields (ACF)) as is already the case with the "Display Conditions" feature.

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by toineenzo** [comment]
You can do this with Post Custom Field

**Reply by roboes** [reply]
Thanks - indeed using "Post Custom Field" and manually typing the custom field variable works. It would be great if Elementor would be able to show a list of custom field variables as it already occurs with "Display Conditions".

**Reply by anton-dealmeida** [reply]
Ah, you're saying they should basically index the fields and have a autocomplete or suggestion feature similar to what they have when ACF Fields are being considered?
